vm.bib

```diff
@@ -0,0 +1,702 @@
+  vm:concepts = {Interpretation},
+  vm:edge:extends = {Brunthaler:2010:EIU + AST interpreters}
+}
+
+@inproceedings{Bolz:2013:SSC,
+  author = {Bolz, Carl Friedrich and Diekmann, Lukas and Tratt, Laurence},
+  booktitle = {Proceedings of the 2013 ACM SIGPLAN International Conference on Object Oriented Programming Systems Languages \& Applications},
+  description = {Storage strategies for collections in dynamically typed languages},
+  doi = {10.1145/2509136.2509531},
+  isbn = {978-1-4503-2374-1},
+  numpages = {16},
+  pages = {167--182},
+  publisher = {ACM},
+  series = {OOPSLA'13},
+  title = {Storage Strategies for Collections in Dynamically Typed Languages},
+  year = 2013,
+  url = "https://tratt.net/laurie/research/pubs/html/bolz_diekmann_tratt__storage_strategies_for_collections_in_dynamically_typed_languages/",
+
+  vm:concepts = {Collections Representation},
+  vm:shortTitle = {Storage Strategies for Collections},
+  vm:edge:extends = {CollectionsRepresentation + adapt representation dynamically to unboxed values}
+}
+
+@inproceedings{Woss:14:TOM,
+  abstract = {Truffle is a Java-based framework for developing high-performance language runtimes. Language implementers aiming at developing new runtimes have to design all the runtime mechanisms for managing dynamically typed objects from scratch. This not only leads to potential code duplication, but also impacts the actual time needed to develop a fully-fledged runtime. In this paper we address this issue by introducing a common object storage model (OSM) for Truffle that can be used by language implementers to develop new runtimes. The OSM is generic, language-agnostic, and portable, as it can be used to implement a great variety of dynamic languages. It is extensible, featuring built-in support for custom extension mechanisms. It is also high-performance, as it is designed to benefit from the optimizing compiler in the Truffle framework. Our initial evaluation indicates that the Truffle OSM can be used to implement high-performance language runtimes, with no performance overhead when compared to language-specific solutions.},
+  author = {Wöß, Andreas and Wirth, Christian and Bonetta, Daniele and Seaton, Chris and Humer, Christian and Mössenböck, Hanspeter},
+  booktitle = {Proceedings of the 2014 International Conference on Principles and Practices of Programming on the Java Platform: Virtual Machines, Languages, and Tools},
+  doi = {10.1145/2647508.2647517},
+  isbn = {978-1-4503-2926-2},
+  numpages = {12},
+  pages = {133--144},
+  publisher = {ACM},
+  series = {PPPJ'14},
+  title = {An Object Storage Model for the Truffle Language Implementation Framework},
+  year = 2014,
+  url = "https://chrisseaton.com/rubytruffle/pppj14-om/pppj14-om.pdf",
+
+  vm:shortTitle = {Truffle Object Model},
+  vm:edge:extends = {
+    Chambers:89:Self + unboxed values, read-only & type specialization;
+    ObjectRepresentation + efficient access and lower memory footprint}
```

Comment:
This extends should be for SELF Maps, not TOSM.

Reply:
Fixed